music_assistant/server/providers/soundcloud/soundcloudpy/asyncsoundcloudpy.py

```diff
@@ -0,0 +1,343 @@
+"""Asyncio.
```

Comment:
I think you want to mention a title here like "Async helpers for connecting to the Soundcloud API" or something along those lines.